test/images/agnhost/netexec/netexec.go

```diff
@@ -339,6 +341,32 @@ func healthzHandler(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusPreconditionFailed)
 }
 
+// readyzHandler response with a 200 if the UDP server is ready. It serves as a readyz that will return a 503
+// once a sig-term has been received.   This allows for graceful removal from endpoints during a pod delete flow.
+func readyzHandler(termCh chan os.Signal) func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		log.Printf("GET /readyz")
+
+		select {
+		case <-termCh:
+			w.WriteHeader(http.StatusServiceUnavailable)
+			if _, err := w.Write([]byte("shutting down")); err != nil {
+				utilruntime.HandleError(err)
+			}
+			return
+
+		default:
+			if serverReady.get() {
```

Comment:
if we set the `serverReady.get()` variable as unready based on the term signal, will that not work for both healthz and readyz handler and simplify the code here?